File: src/main/java/us/n8l/duplicatefinder/digests/SHA1.java
package us.n8l.duplicatefinder.digests;

import org.jetbrains.annotations.NotNull;

public class SHA1 extends BaseDigest {
    SHA1(@NotNull byte[] bytes) {
        super(bytes);
    }

    @Override
    protected Algorithm getAlgorithm() {
        return Algorithm.MD5;
    }
}

File: src/main/java/us/n8l/duplicatefinder/digests/SHA1.kt
package us.n8l.duplicatefinder.digests

import java.io.InputStream

class SHA1 : BaseDigest {
    internal constructor(bytes: ByteArray) : super(bytes)

    internal constructor(inStrean: InputStream) : super(inStrean)

    override fun getAlgorithm() = Algorithm.MD5
}
